src/cljs/ataru/virkailija/application/view.cljs

```diff
@@ -108,11 +108,26 @@
   (let [events (subscribe [:state-query [:application :events]])]
     (fn []
       (into
-        [:div [:div.application-handling__review-header "Tapahtumat"]]
+        [:div.application-handling__event-list
+         [:div.application-handling__review-header "Tapahtumat"]]
         (mapv #(event-row %) @events)))))
 
+(defn application-review-notes []
+  (let [review (subscribe [:state-query [:application :review]])
+        ; React doesn't like null, it leaves the previous value there, hence:
+        review->notes-str (fn [review] (let [notes (:notes @review)] (if notes notes "")))]
```

Comment:
Then I'd return nil, but:
React doesn't like null, it leaves the previous value there, hence: